recipes/uriparser/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "uriparser" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: 3ea184e1ef2286e54ce712364780f1b29d1c58233f7a142b11c0550a910e295c
+
+build:
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+
+test:
+  commands:
+    - test -f $PREFIX/lib/liburiparser.a                            # [not win]
+    - if not exist %PREFIX%\\Library\\lib\\uriparser.lib exit 1     # [win]
+
+about:
+  home: https://uriparser.github.io/
+  license: BSD-3-Clause
+  license_family: BSD
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: COPYING
+  summary: 'RFC 3986 compliant URI parsing and handling library written in C89'
+
+  # The remaining entries in this section are optional, but recommended.
+  description: |
+    uriparser is a strictly RFC 3986 compliant URI parsing and handling
+    library written in C89 ("ANSI C"). uriparser is cross-platform, fast,
+    supports Unicode, and is licensed under the New BSD license.
+  doc_url: https://uriparser.github.io/
+  dev_url: https://github.com/uriparser/uriparser
+
+extra:
+  recipe-maintainers:
+    - pitrou
+    - wesm
```

Comment:
@wesm are you OK with being a maintainer here?

Reply:
yes

Reply:
yes